erpnext/selling/doctype/sales_order/sales_order_list.js

```diff
@@ -15,18 +15,24 @@ frappe.listview_settings['Sales Order'] = {
 			// not delivered & overdue
 				return [__("Overdue"), "red",
 					"per_delivered,<,100|delivery_date,<,Today|status,!=,Closed"];
+			} else if (doc.docstatus == 1 && doc.per_picked == 100 && doc.per_delivered < 100) {
+				return [__("To Pick"), "orange",
+					"per_picked,<,100|per_billed,=,100|status,!=,Closed"];
```

Comment:
Add `per_delivered` here too

Reply:
per_delivered or per_billed?

Reply:
On lines 19-20, I mean. It won't add the `per_delivered` filter

Reply:
okay understood

Reply:
@MyuddinKhatri, I think you missed this one